mist/runtime/soft_skeleton.py

```diff
@@ -0,0 +1,50 @@
+
+import torch
+import torch.nn.functional as F
```

Comment:
For the sake of consistency, maybe add 

import torch.nn as nn

Also please rename this file to something like loss_utils.py.